os/kernel/binary_manager/binary_manager.h

```diff
@@ -156,12 +159,15 @@ typedef struct statecb_node_s statecb_node_t;
 binmgr_uinfo_t *binary_manager_get_udata(uint32_t bin_idx);
 #define BIN_ID(bin_idx)                                 binary_manager_get_udata(bin_idx)->bin_id
 #define BIN_STATE(bin_idx)                              binary_manager_get_udata(bin_idx)->state
+#define BIN_USEIDX(bin_idx)                             binary_manager_get_udata(bin_idx)->useidx
 #define BIN_RTLIST(bin_idx)                             binary_manager_get_udata(bin_idx)->rt_list
 #define BIN_NRTLIST(bin_idx)                            binary_manager_get_udata(bin_idx)->nrt_list
 
-#define BIN_VER(bin_idx)                                binary_manager_get_udata(bin_idx)->bin_ver
+#define BIN_VER(bin_idx)                                binary_manager_get_udata(bin_idx)->load_attr.bin_ver
 #define BIN_KERNEL_VER(bin_idx)                         binary_manager_get_udata(bin_idx)->kernel_ver
 #define BIN_CBLIST(bin_idx)                             binary_manager_get_udata(bin_idx)->cb_list
+#define BIN_FILECNT(bin_idx)                            binary_manager_get_udata(bin_idx)->file_cnt
+#define BIN_FILEVER(bin_idx, file_idx)                  binary_manager_get_udata(bin_idx)->bin_ver[file_idx]
```

Comment:
I think this is not file version. This is binary version. And we don't have any rule for file version.
We'd better rename like `BIN_VER` as `BIN_LATEST_VER`, and `BIN_FILEVER` as `BIN_VER`. Because active(running) binary will be always the latest one.